### PR TITLE
Markdown and propstable are their own components

### PR DESCRIPTION
**Pull request by Matt-Butler**
### Summary
Updating with markdown and propsTable now being their own components in the monorepo.

@cerner/terra

### CODE REVIEW

File: generators/app/templates/package.json [hunk 1]
```diff
@@ -31,7 +31,9 @@
   "dependencies": {
     "classnames": "^2.2.5",
     "react": "^15.4.2",
-    "terra-mixins": "^1.0.0"
+    "terra-mixins": "^1.0.0",
+    "terra-props-table": "0.1.0",
+    "terra-markdown": "0.1.0",
```
Comment (StephenEsser) [inline generators/app/templates/package.json]: Although I'm not sure why travis CI didn't have a problem with this, running locally this trailing comma in the package.json fails on npm install

Comment (StephenEsser) [inline generators/app/templates/package.json]: ( When generating a project and npm installing from terra-ui )

Comment (bjankord) [inline generators/app/templates/package.json]: Good catch, yeah this needs updated.

Comment (Matt-Butler) [inline generators/app/templates/package.json]: https://github.com/cerner/generator-terra-module/pull/21/commits/fcac95d7f9b6ddeaf75191cfb28ac94f54d4ba8b

**Pull request merged**